pkg/whitesource/whitesource.go

```diff
@@ -115,24 +126,42 @@ func (s *System) GetProjectsMetaInfo(productToken string) ([]Project, error) {
 }
 
 //GetProjectToken returns the project token for a project with a given name
-func (s *System) GetProjectToken(productToken, projectName string) (string, error) {
+func (s *System) GetProjectByName(productToken, projectName string) (*Project, error) {
```

Comment:
where did the function `GetProjectToken()` go?
Seems like an incompatible change

Reply:
Was resolved it seems.